diff --git a/dist/factory/program.js b/dist/factory/program.js
index 0a39b01cf9b06c837326c982ec5ce46908bc876b..d07197b93d4c685ca5542ab723ddbb2977494d71 100644
--- a/dist/factory/program.js
+++ b/dist/factory/program.js
@@ -6,7 +6,7 @@ const path = tslib_1.__importStar(require("node:path"));
 const normalize_path_1 = tslib_1.__importDefault(require("normalize-path"));
 const typescript_1 = tslib_1.__importDefault(require("typescript"));
 const Errors_js_1 = require("../src/Error/Errors.js");
-const node_fs_1 = tslib_1.__importDefault(require("node:fs"));
+const glob_1 = require("glob");
 function loadTsConfigFile(configFile) {
     const raw = typescript_1.default.sys.readFile(configFile);
     if (!raw) {
@@ -54,7 +54,7 @@ function getTsConfig(config) {
 }
 function createProgram(config) {
     const rootNamesFromPath = config.path
-        ? node_fs_1.default.globSync((0, normalize_path_1.default)(path.resolve(config.path))).map((rootName) => (0, normalize_path_1.default)(rootName))
+        ? (0, glob_1.globSync)((0, normalize_path_1.default)(path.resolve(config.path))).map((rootName) => (0, normalize_path_1.default)(rootName))
         : [];
     const tsconfig = getTsConfig(config);
     const rootNames = rootNamesFromPath.length ? rootNamesFromPath : tsconfig.fileNames;
